massa-async-pool/src/changes.rs

```diff
@@ -51,4 +51,16 @@ impl AsyncPoolChanges {
     pub fn push_delete(&mut self, msg_id: AsyncMessageId) {
         self.0.push(Change::Delete(msg_id));
     }
+
+    /// Retrieve only the added messages
+    pub fn get_add(&self) -> Vec<(AsyncMessageId, AsyncMessage)> {
```

Comment:
```suggestion
    pub fn get_adds(&self) -> Vec<(AsyncMessageId, AsyncMessage)> {
```

Because the function gets multiple add messages

Reply:
@AurelienFT Not a big fan of `get_adds` what about `get_added_messages`?

Reply:
I prefer also